megamek/src/megamek/common/LandAirMech.java

```diff
@@ -127,6 +127,7 @@ public String[] getLocationAbbrs() {
     private int altLoss = 0;
     private int altLossThisRound = 0;
 
+
```

Comment:
Was this added intentionally?

Reply:
Regardless of the answer to this question, it's surely more effort than it's worth it to worry about a single blank line

Reply:
It's nearly no effort to fix, and everyone should get in the habit of spotting unintended changes to their PRs as a matter of course.

Reply:
My bad, fixed in c089690